### DISCUSSION TITLE
Why it remind me the message?

### DISCUSSION THREAD

**Discussion by LightToDawn** [fraxiinus/ReplayBook#192] [Troubleshooting]
![image](https://user-images.githubusercontent.com/93030018/172981540-e414ef85-714a-4cc0-a05c-49599c864637.png)
![image](https://user-images.githubusercontent.com/93030018/172981638-3de9f417-400b-489a-9ce3-96b265dfc19a.png)

**Comment by LightToDawn** [comment]
by the way,when I use 6.0.5 version for the .net,it still remind me this message

**Reply by fraxiinus** [reply]
Try getting the 6.0.5 **Desktop** Runetime

https://dotnet.microsoft.com/en-us/download/dotnet/6.0
![image](https://user-images.githubusercontent.com/8365233/172982940-52bf81a0-11aa-4ace-873f-b3e4889a41c3.png)

**Reply by LightToDawn** [reply]
> Try getting the 6.0.5 **Desktop** Runetime
> 
> https://dotnet.microsoft.com/en-us/download/dotnet/6.0 ![image](https://user-images.githubusercontent.com/8365233/172982940-52bf81a0-11aa-4ace-873f-b3e4889a41c3.png)

oh,I had solve it,thank you!
